public/index.html

```diff
@@ -9,11 +9,11 @@
     <!-- Lato regular, regular italic -->
     <link href="https://fonts.googleapis.com/css2?family=Lato:ital@0;1&display=swap" rel="stylesheet"> 
 
-    <title><%= htmlWebpackPlugin.options.title %></title>
+    <title>UBC Launchpad</title>
```

Comment:
*officially*, this is two words ;) 

```suggestion
    <title>UBC Launch Pad</title>
```